src/test/java/service/ResourceServiceTest.java

```diff
@@ -0,0 +1,19 @@
+package service;
+
+import org.junit.jupiter.api.Test;
+import webserver.request.RequestLine;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ResourceServiceTest {
+
+    @Test
+    void canServe() {
+        RequestLine requestLine = RequestLine.from("GET /user/login.html HTTP/1.1");
+
+        ResourceService resourceService = new ResourceService();
+        boolean b = resourceService.canServe(requestLine);
+
+        assertThat(b).isTrue();
```

Comment:
```suggestion
        assertThat(resourceService.canServe(requestLine)).isTrue();
```

아무리 간단한 코드라 하더라도, 한개 짜리 변수명은 허용하지 않는게 좋을 것 같아요.
차라리 익명으로 처리하는 건 어떠신가요?